cms/djangoapps/maintenance/views.py

```diff
@@ -0,0 +1,217 @@
+"""
+Views for the maintenance app.
+"""
+import logging
+from django.db import transaction
+from django.core.urlresolvers import reverse_lazy
+from django.utils.decorators import method_decorator
+from django.utils.translation import ugettext_lazy as _
+from django.views.generic import View
+
+from edxmako.shortcuts import render_to_response
+from opaque_keys import InvalidKeyError
+from opaque_keys.edx.keys import CourseKey
+from xmodule.modulestore.django import modulestore
+from xmodule.modulestore.exceptions import ItemNotFoundError
+
+from contentstore.management.commands.utils import get_course_versions
+from util.views import require_global_staff
+
+
+log = logging.getLogger(__name__)
+
+MAINTENANCE_COMMANDS = {
```

Comment:
Actually, Index view lists down all the commands along with their name and description and needs URL to forward to the specific view. 

This also helps in getting the list of all the URLs in the maintenance app which is used by many tests.